src/server/bloomcache/src/main.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+#[macro_use]
+extern crate logger;
+
+use backtrace::Backtrace;
+use clap::{App, Arg};
+use config::BloomcacheConfig;
+use pelikan_bloomcache::BloomCache;
+use rustcommon_metrics::*;
+use server::PERCENTILES;
+
+fn main() {
+    // custom panic hook to terminate whole process after unwinding
+    std::panic::set_hook(Box::new(|s| {
+        error!("{}", s);
```

Comment:
Change this to eprintln please. We don't guarantee the log is flushed before terminating and having the error message lost is not a great experience.